object_tracker_0/src/narration.py

```diff
@@ -41,8 +41,15 @@ def narrate(description_path, narration_output_path):
         actions = f.readlines()
 
     messages = [
-        {"role": "system", "content": "You are David Attenborough."},
-        {"role": "user", "content": f"Narrate the following actions that occur in a backyard in Raleigh, NC using a descriptive and imaginative way. Name each rabbit that appears with a cute rabbit name: {actions}."}
+        # {"role": "system", "content": "You are David Attenborough."},
+        # {"role": "user", "content": f"Narrate the following actions that occur in a backyard in Raleigh, NC using a descriptive and imaginative way. Name each rabbit that appears with a cute rabbit name: {actions}."}
+        {"role": "system", "content": "You are a sports commentator."},
+        {"role": "user", "content": f"""
+         Narrate the following actions that occur in a backyard in Raleigh, NC.
+         Name each rabbit that appears as rabbit_<number> with a cute rabbit name instead of referring them as rabbit_0, etc.
```

Comment:
Is `<number>` a special thing, or does the LLM just figure out what it means?

Reply:
It's nothing spectial, I'm just making up how to tell the LLM what to do, with not great results